### DISCUSSION TITLE
keep current path when press [alt+T] or [alt+N]

### DISCUSSION THREAD

**Discussion by myhuman** [wavetermdev/waveterm#1043] [General]
Current I'm focus on a terminal tab, with work folder "~/develop/linux/driver".
When I press  [alt+T] or [alt+N], the work folder of the new one is home.

I'd like to keep the work folder when new tab open.
How can I do?

**Comment by esimkowitz** [comment]
ah interesting, yeah should be easy enough to make a setting. Can you open a feature request?

**Comment by myhuman** [comment]
https://github.com/wavetermdev/waveterm/issues/1130
